src/modules/emotion-list.json

```diff
@@ -0,0 +1,4 @@
+{
+    "text": {},
```

Comment:
dont keep here, need use i18n instead for texts

Reply:
afaik you had plans to restructure i18n
It's confusing to use i18n here now =\

Reply:
why? csv file great... but need several files for better working =) i push upd about what

Reply:
That's what I'm talking about